Kinvey/Kinvey/Cache.swift

```diff
@@ -11,6 +11,7 @@ import Foundation
 internal protocol CacheType: class {
     
     var ttl: TimeInterval? { get set }
+    var lastPull: Date? { get set }
```

Comment:
The difficulty with this API is if the developer uses queries. We need to maintain `lastPull` by query.